go/pserver/service.go

```diff
@@ -26,6 +27,8 @@ const (
 	AlreadyInitialized = "pserver already initialized"
 	// Uninitialized is true if pserver not fully initialized
 	Uninitialized = "pserver not fully initialized"
+	// CheckpointUnmatched is true if the md5 of checkpoint file does matched the older
```

Comment:
I think "is true" is not very appropriate in English grammar, it's `string` type, "is true" is for `bool` type. Since the content of these constants already express the meaning, maybe we don't need the comment?
Perhaps we can put a comment on the top of `const` keyword:
```Go
// RPC error messages.
const (
  AlreadyInitialized = "pserver already initialized"
  Uninitialized = "pserver not fully initialized"
  CheckpointUnmatched = "checkpoint file does not matched the older"
)
```